src/Decompiler/Loading/Loader.cs

```diff
@@ -83,47 +86,83 @@ public Program AssembleExecutable(string fileName, byte[] image, IAssembler asm,
         }
 
         /// <summary>
-        /// Loads the image into memory, unpacking it if necessary. Then, relocate the image.
-        /// Relocation gives us a chance to determine the addresses of interesting items.
+        /// Loads (or assembles) a Reko decompiler project. If a binary file is
+        /// specified instead, we create a simple project for the file. If an archive
+        /// is encountered, return that.
         /// </summary>
-        /// <param name="rawBytes">Image of the executable file.</param>
+        /// <param name="imageLocation">The location of the image to load.</param>
+        /// <param name="loaderName">Optional .NET class name of a custom
+        /// image loader</param>
+        /// <param name="addrLoad">Optional address at which to load the image.
+        /// </param>
+        /// <returns>An instance of <see cref="ILoadedImage"/> if an image could be loaded, 
+        /// or null otherwise.</returns>
```

Comment:
`an instance of <see cref="Blob"/> otherwise`

Reply:
Fixed.